src/main/java/com/gd/clinic/entities/Prescription.java

```diff
@@ -0,0 +1,40 @@
+package com.gd.clinic.entities;
+
+import lombok.*;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.time.OffsetDateTime;
+import java.util.UUID;
+
+@NoArgsConstructor
+@RequiredArgsConstructor
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "Prescriptions")
+public class Prescription {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Type(type = "uuid-char")
+    private UUID id;
+
+    @NonNull
+    @Type(type = "uuid-char")
+    private UUID patientId;
+
+    @NonNull
+    @Type(type = "uuid-char")
+    private UUID treatmentId;
+
+    @NonNull
+    String timePattern;
+
+    @NonNull
+    Integer period;
+
+    OffsetDateTime datePrescribed = OffsetDateTime.now();
```

Comment:
this value should be set in the service layer

Reply:
Ok, will do that from now on. The service layer was created on previous commit and the value is now set in `com.gd.clinic.service.PrescriptionService`